sdk/metricsadvisor/ai-metrics-advisor/sample.env

```diff
@@ -24,3 +24,5 @@ METRICS_ADVISOR_API_KEY=<api key>
 # METRICS_ADVISOR_AZURE_BLOB_DETECTION_CONFIG_ID=
 # METRICS_ADVISOR_AZURE_SQLSERVER_DETECTION_CONFIG_ID=
 # METRICS_ADVISOR_AZURE_SQLSERVER_DETECTION_INCIDENT_ID=
+# METRICS_ADVISOR_AZURE_SQLSERVER_DATAFEED_ID=
```

Comment:
do we still use blob datafeed id, blob metric id 1, and blob detection config id? I don't see removal here

Reply:
Removed it